otterdog/eclipse-tractusx.jsonnet

```diff
@@ -302,6 +302,27 @@ orgs.newOrg('eclipse-tractusx') {
         },
       ],
     },
+    orgs.newRepo('tx-profiles') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      has_discussions: true,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
in the long run we would like to enforce only read permissions by default. While it is quite some work for existing repo to ensure that all their workflows still work, for new repos I would suggest to already start with read permissions. WDYT?

Reply:
since this repo is supposed to mainly contain files (no code), so i don't anticipate any sophisticated workflows. But what would be the practical consequence? could I still request `write` permission per-workflow, in the workflow file?

Reply:
Indeed, everything would still be working, but you need to explicitly request write tokens in the workflow file itself:

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

So its an effort to declare what permissions the workflow really needs instead of granting it write tokens by default which could be misused by 3rd party actions.

Reply:
that totally makes sense :) updated the PR